packages/framework-integration-tests/integration/end-to-end/cart.integration.ts

```diff
@@ -0,0 +1,155 @@
+import { ApolloClient } from 'apollo-client'
+import { NormalizedCacheObject } from 'apollo-cache-inmemory'
+import { graphQLClient, waitForIt } from '../providers/aws/utils'
+import { random } from 'faker'
+import { expect } from 'chai'
+import gql from 'graphql-tag'
+import { CartItem } from '../../src/common/cart-item'
+
+describe('Cart end-to-end tests', () => {
+  let client: ApolloClient<NormalizedCacheObject>
+
+  before(async () => {
+    client = await graphQLClient()
+  })
+
+  describe('Commands', () => {
+    it('accepts a command successfully', async () => {
+      const response = await client.mutate({
+        variables: {
+          cartId: random.uuid(),
+          productId: random.uuid(),
+          quantity: random.number({ min: 1 }),
+        },
+        mutation: gql`
+          mutation ChangeCartItem($cartId: ID!, $productId: ID!, $quantity: Float) {
+            ChangeCartItem(input: { cartId: $cartId, productId: $productId, quantity: $quantity })
+          }
+        `,
+      })
+
+      expect(response).not.to.be.null
+      expect(response?.data?.ChangeCartItem).to.be.true
+    })
+
+    describe('Read models', () => {
```

Comment:
Was this `describe('Read models')` intended to be placed inside the `describe('Commands')`? It seems there is no relation between them

Reply:
it was intended to reflect that a Command makes changes in a read model. In my head, I was thinking that a Command in the future might have an impact over another thing that not necessarily has to be a read model.